elasticassignment/ageofempires/models/unit.py

```diff
@@ -0,0 +1,18 @@
+import ageofempires.ElasticClient.Client as Client
```

Comment:
If you check all your model classes, there is repeated code which means all the model classes are performing same kind of operations (which is expected from them)!

However, this kind of similar classes give us the opportunity to implement a basic OOP concept. What's that and how will you implement it? (Please discuss before making changes)